arangod/Agency/MoveShard.cpp

```diff
@@ -519,11 +534,19 @@ JOB_STATUS MoveShard::pendingLeader() {
     doForAllShards(_snapshot, _database, shardsLikeMe,
                    [this, &done](Slice plan, Slice current,
                                  std::string& planPath, std::string& curPath) {
-                     for (VPackSlice s : VPackArrayIterator(current)) {
-                       if (s.copyString() == _to) {
-                         ++done;
-                         return;
```

Comment:
I would not remove this `return` statement.

Reply:
WTF? I did not actively remove it.

Reply:
yes